engine/installation/linux/ubuntu.md

```diff
@@ -101,6 +101,11 @@ Docker from the repository.
 
     To disable the `testing` repository, you can edit `/etc/apt/sources.list`
     and remove the word `testing` from the appropriate line in the file.
+    
+    > **Note**: Sometimes, in a distribution like Linux Mint, you might have
+    > to change **ubuntu-$(lsb_release -cs)** to your parent Ubuntu distribution.
```

Comment:
Please use backticks instead of asterisks, so this will be monotype instead of bold. Thanks!